File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/7 - 9 and 10 March 2019/05. Fitness Center.py
people = int(input())

count_back = 0
count_chest = 0
count_legs = 0
count_abs = 0
count_protein_shake = 0
count_protein_bar = 0

count_workout = 0
count_protein_use = 0

for i in range(people):
    command = input()
    if command == "Back":
        count_back += 1
        count_workout += 1
    elif command == "Chest":
        count_chest += 1
        count_workout += 1
    elif command == "Legs":
        count_legs += 1
        count_workout += 1
    elif command == "Abs":
        count_abs += 1
        count_workout += 1
    elif command == "Protein shake":
        count_protein_shake += 1
        count_protein_use += 1
    elif command == "Protein bar":
        count_protein_bar += 1
        count_protein_use += 1

print(f"{count_back} - back")
print(f"{count_chest} - chest")
print(f"{count_legs} - legs")
print(f"{count_abs} - abs")
print(f"{count_protein_shake} - protein shake")
print(f"{count_protein_bar} - protein bar")
print(f"{(count_workout / people * 100):.2f}% - work out")
print(f"{(count_protein_use / people * 100):.2f}% - protein")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/03. Online Exam - 9 and 10 March 2019/05. Fitness Center.js
function fitnessCenter(input) {
    // Mask
    let ac = {
        'Back': 0,
        'Chest': 0,
        'Legs': 0,
        'Abs': 0,
        'Protein shake': 0,
        'Protein bar': 0
    };

    let acts = Number(input.shift());

    for (let act = 1; act <= acts; act += 1) {
        ac[input.shift()] += 1;
    }

    let training = ac['Back'] + ac['Chest'] + ac['Legs'] + ac['Abs'];
    let suppliments = ac['Protein shake'] + ac['Protein bar'];

    console.log(`${ac['Back']} - back`);
    console.log(`${ac['Chest']} - chest`);
    console.log(`${ac['Legs']} - legs`);
    console.log(`${ac['Abs']} - abs`);
    console.log(`${ac['Protein shake']} - protein shake`);
    console.log(`${ac['Protein bar']} - protein bar`);
    console.log(`${(training / acts * 100).toFixed(2)}% - work out`);
    console.log(`${(suppliments / acts * 100).toFixed(2)}% - protein`);
}

fitnessCenter([10, 'Back', 'Chest', 'Legs', 'Abs', 'Protein shake', 'Protein bar', 'Protein shake', 'Protein bar', 'Legs', 'Abs']);  
// Should return:
// 1 - back
// 1 - chest
// 2 - legs
// 2 - abs
// 2 - protein shake
// 2 - protein bar
// 60.00% - work out
// 40.00% - protein

fitnessCenter([7, 'Chest', 'Back', 'Legs', 'Legs', 'Abs', 'Protein shake', 'Protein bar']);  
// Should return:
// 1 - back
// 1 - chest
// 2 - legs
// 1 - abs
// 1 - protein shake
// 1 - protein bar
// 71.43% - work out
// 28.57% - protein
